Tests/Acceptance/ID3TagEditorTestAcceptanceNewFrames.swift

```diff
@@ -208,21 +224,28 @@ class ID3TagEditorTestAcceptanceNewFrames: XCTestCase {
                 .UserDefinedTextInformation : ID3FrameUserDefinedText(description: "description", content: "UserDefinedTextInformation V2"),
             ]
         )
-        
+
         XCTAssertNoThrow(try id3TagEditor.write(
             tag: id3Tag,
             to: PathLoader().pathFor(name: "example-newframes", fileType: "mp3"),
             andSaveTo: pathMp3Generated
             ))
+        let generated = try Data(contentsOf: URL(fileURLWithPath: pathMp3Generated))
+        let expected = try Data(contentsOf: URL(fileURLWithPath: pathMp3ToCompare))
+        #warning("We are comparing against the same corrupted file from the read tests.")
```

Comment:
Yeah, the practice of writing the file using the same function that the output would be compared against was written with wasn't ideal, but I think anything else caused the test to fail because the file sizes didn't match, even though the tests of the content all passed.

For instance, the first time I tried to generate a written file to compare the test output to, I did it in using the single-button interface I created to test writing in AudiobookTagger, which was in fact written by ID3TagEditor, but the file sizes were ever so slightly off. I couldn't think of any other way to generate a comparison file thay would be sure to have all the same frames in all the same places, because every metadata writer handles them a bit differently.

Reply:
Yes, its fine to use such a file for a test. You just have to know you have it working properly first. At the moment this test is making sure we generate a corrupt file.

To break the circular reasoning, the steps to this are:

1. Make sure it correctly reads a valid file created from a different trusted source.

2. Make sure what it writes can be read by the same code.